torchmultimodal/utils/common.py

```diff
@@ -133,6 +133,19 @@ def transpose_for_scores(
     return x.permute(0, 2, 1, 3)
 
 
+def load_module_from_url(m: torch.nn.Module, url: str, progress=True):
```

Comment:
This function duplicates what `PretrainedMixin` does? Is it possible to also refactor `PretrainedMixin` such that `load_model` would call the utility?

Reply:
yosua can u just create a task and send me. we should try moving to this util for other models if possible and deprecate the mixin

Reply:
nit : can u change m to "model"

Reply:
I will change `m` to be `module`, and will change `model_dir` to `module_dir` as well. Because I think the term `module` is more generic and suitable for this case.